ballista/rust/core/Cargo.toml

```diff
@@ -40,7 +40,7 @@ tokio = "1.0"
 tonic = "0.4"
 uuid = { version = "0.8", features = ["v4"] }
 
-arrow-flight = { version = "4.0"  }
+arrow-flight = { git = "https://github.com/apache/arrow-rs.git", rev = "e5cda312b697c3d610637b28c58b6f1b104b41cc" }
```

Comment:
because this change relies on https://github.com/apache/arrow-datafusion/pull/520/files#diff-d76aa4d1931f29ba8725c7d31887fb45c5d8d74a026dae914e2012be1a15be33R28

Reply:
For this we need to wait on 4.3.0. this includes the required changes, no?

https://github.com/apache/arrow-rs/pull/444

Reply:
> For this we need to wait on 4.3.0. this includes the required changes, no?
> 
> 
> 
> https://github.com/apache/arrow-rs/pull/444
> 
> 

Yes it does and after the crate version is up i will remove the changes to cargo.toml files